pkg/middleware/region_proxy.go

```diff
@@ -6,33 +6,64 @@ import (
 	"net/http"
 	"net/http/httputil"
 	"net/url"
+	"strings"
+	"time"
 
+	"github.com/hashicorp/golang-lru/v2/expirable"
 	sdk "github.com/openshift-online/ocm-sdk-go"
 	"github.com/openshift-online/ocm-sdk-go/logging"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
+	// region ids
+	APSoutEast1 = "ap-southeast-1"
+	USEast1     = "us-east-1"
+	Global      = "global"
+
+	cacheSize       = 1000
+	cacheExpireTime = time.Hour * 24
```

Comment:
these default values are fine but they should probably be in RegionProxyMiddwareOption in case CS/OSL find the need to tweak them